internal/connector/pingone/platform/resources/pingone_notification_template_content.go

```diff
@@ -15,152 +15,129 @@ var (
-				templateLocale, templateLocaleOk = templateContents.TemplateContentPush.GetLocaleOk()
-				templateVariant, templateVariantOk = templateContents.TemplateContentPush.GetVariantOk()
-			case templateContents.TemplateContentSMS != nil:
-				templateContentsId, templateContentsIdOk = templateContents.TemplateContentSMS.GetIdOk()
-				templateDeliveryMethod, templateDeliveryMethodOk = templateContents.TemplateContentSMS.GetDeliveryMethodOk()
-				templateLocale, templateLocaleOk = templateContents.TemplateContentSMS.GetLocaleOk()
-				templateVariant, templateVariantOk = templateContents.TemplateContentSMS.GetVariantOk()
-			case templateContents.TemplateContentVoice != nil:
-				templateContentsId, templateContentsIdOk = templateContents.TemplateContentVoice.GetIdOk()
-				templateDeliveryMethod, templateDeliveryMethodOk = templateContents.TemplateContentVoice.GetDeliveryMethodOk()
-				templateLocale, templateLocaleOk = templateContents.TemplateContentVoice.GetLocaleOk()
-				templateVariant, templateVariantOk = templateContents.TemplateContentVoice.GetVariantOk()
+			case templateContent.TemplateContentPush != nil:
+				templateContentId, templateContentIdOk = templateContent.TemplateContentPush.GetIdOk()
+				templateContentDeliveryMethod, templateContentDeliveryMethodOk = templateContent.TemplateContentPush.GetDeliveryMethodOk()
+				templateContentLocale, templateContentLocaleOk = templateContent.TemplateContentPush.GetLocaleOk()
+				templateContentVariant = templateContent.TemplateContentPush.GetVariant()
+			case templateContent.TemplateContentSMS != nil:
+				templateContentId, templateContentIdOk = templateContent.TemplateContentSMS.GetIdOk()
+				templateContentDeliveryMethod, templateContentDeliveryMethodOk = templateContent.TemplateContentSMS.GetDeliveryMethodOk()
+				templateContentLocale, templateContentLocaleOk = templateContent.TemplateContentSMS.GetLocaleOk()
+				templateContentVariant = templateContent.TemplateContentSMS.GetVariant()
+			case templateContent.TemplateContentEmail != nil:
+				templateContentId, templateContentIdOk = templateContent.TemplateContentEmail.GetIdOk()
+				templateContentDeliveryMethod, templateContentDeliveryMethodOk = templateContent.TemplateContentEmail.GetDeliveryMethodOk()
+				templateContentLocale, templateContentLocaleOk = templateContent.TemplateContentEmail.GetLocaleOk()
+				templateContentVariant = templateContent.TemplateContentEmail.GetVariant()
+			case templateContent.TemplateContentVoice != nil:
+				templateContentId, templateContentIdOk = templateContent.TemplateContentVoice.GetIdOk()
+				templateContentDeliveryMethod, templateContentDeliveryMethodOk = templateContent.TemplateContentVoice.GetDeliveryMethodOk()
+				templateContentLocale, templateContentLocaleOk = templateContent.TemplateContentVoice.GetLocaleOk()
+				templateContentVariant = templateContent.TemplateContentVoice.GetVariant()
 			default:
 				continue
```

Comment:
Should this error or warn?

Reply:
I think a warning here might be appropriate, and this is not considered to be expected nor normal